app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,25 @@
+class CommentsController < ApplicationController
+  def create
+    @comment = current_user.comments.new(comment_params)
```

Comment:
Kindly define a `new` action/method for this controller. Your line 3 uses the new method which is currently undefined.

Consider this snippet.

```ruby
def new
   # code
end
```